app/components/ui/molecules/DashboardListItem.js

```diff
@@ -85,12 +85,8 @@ const DashboardListItem = ({ statusCode, title, date }) => (
 
 DashboardListItem.propTypes = {
   statusCode: PropTypes.string.isRequired,
-  title: PropTypes.string,
+  title: PropTypes.string.isRequired,
```

Comment:
Does this cause any problems for submissions before they've reached the upload manuscript stage? Is there no title data present or does gql return an empty string on the index?